etcdmain/config.go

```diff
@@ -293,6 +304,9 @@ func (cfg *config) Parse(arguments []string) error {
 	if 5*cfg.TickMs > cfg.ElectionMs {
 		return fmt.Errorf("-election-timeout[%vms] should be at least as 5 times as -heartbeat-interval[%vms]", cfg.ElectionMs, cfg.TickMs)
 	}
+	if cfg.ElectionMs > maxElectionMs {
+		return fmt.Errorf("-election-timeout[%vms] is unreasonably high, which is expected to be less than %vms", cfg.ElectionMs, maxElectionMs)
```

Comment:
-election-timeout[%vms] is too long. Set `-election-timeout` less than %vms. See ... for details.